### PR TITLE
set up prisma client

### PR DESCRIPTION
**Pull request by Capn05**
prisma database is all set up, there are routes for each necessary crud operation and user is registered into firebase and prisma database on registration.

### CODE REVIEW

File: Routes/users.js [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+//       email,
+//     },
+//   });
+//   res.json(newUser);
+// });
 
-router.put("/:id", async (req, res) => {
-  const { id } = req.params;
-  const { email } = req.body;
-  const updatedUser = await prisma.user.update({
```
Comment (samyalihamad) [inline Routes/users.js:25]: @Capn05 curious does prisma sanitize the user input?

File: src/login/register.jsx [hunk 2]
```diff
@@ -13,21 +13,21 @@ const Register = () => {
 
   const { userLoggedIn } = useAuth();
 
-  const onSubmit = async (e) => {
-    e.preventDefault();
-    if (!isRegistering) {
```
Comment (samyalihamad) [inline src/login/register.jsx:18]: Is the idea to use `isRegistering` for a loading animation as well?

**Comment by BriptimusPrimus** [comment]
Leaving commented out code is against best practices. In cases like this, the right thing to do is to delete the code.

You can always come back to the PR where the code is deleted to recover any code you need.

File: Routes/users.js (hunk 1 above)
Comment (Capn05) [reply Routes/users.js:25]: Prisma uses prepared statements, preventing sql injection. It is also a type safe API.

File: src/login/register.jsx (hunk 2 above)
Comment (Capn05) [reply src/login/register.jsx:18]: yes!

**Pull request merged**